binaries/conda/build_packages.py

```diff
@@ -62,22 +66,27 @@ def install_miniconda():
         )
         return
 
-    os.system(f"rm -rf $HOME/miniconda")
-    exit_code = os.system(f"wget {MINICONDA_DOWNLOAD_URL} -O ~/miniconda.sh")
-    if exit_code != 0:
-        print(f"miniconda download failed")
-        return exit_code
-    os.system(f"bash ~/miniconda.sh -f -b -p $HOME/miniconda")
-    os.system(f"echo 'export PATH=$HOME/miniconda/bin:$PATH' >> ~/.bashrc")
-    os.system(f"ln -s $HOME/miniconda/bin/activate $HOME/miniconda/condabin/activate")
-    os.system(
-        f"ln -s $HOME/miniconda/bin/deactivate $HOME/miniconda/condabin/deactivate"
+    try_and_handle(f"rm -rf $HOME/miniconda", dry_run)
```

Comment:
How are we verifying this? In the github actions, since conda is already installed, I think this code doesn't execute

Reply:
This was verified in my logs locally